tools/scripts/generate_binary_build_matrix.py

```diff
@@ -339,8 +336,34 @@ def generate_wheels_matrix(
             desired_cuda = translate_desired_cuda(gpu_arch_type, gpu_arch_version)
 
             installation_pypi = ""
-            if os == "linux" and gpu_arch_version == "11.7" and channel != "release":
+            # special 11.7 wheels package without dependencies
+            # dependency downloaded via pip install
+            if arch_version == "11.7" and os == "linux":
                 installation_pypi = get_wheel_install_command(channel, gpu_arch_type, desired_cuda, python_version, True)
```

Comment:
This is special wheel built for pypi upload

Reply:
This is something we want to do _only_ for pytorch right? We don't want for this to be added to every linux / cuda 11.7 build for every library right?

Reply:
yes this is only for pytorch, we don't have a parameter/argument to be able to tell for which system this script is being called for.  Looks like I need to add this parameter with_wheel_cudnn, false by default

Reply:
Added guard against adding this build everywhere. Also changed test PR